sources/tools/Stride.Importer.Assimp/MeshConverter.cs

```diff
@@ -0,0 +1,1375 @@
+                        {
+                            *((ushort*)ibPointer) = (ushort)(mesh->MFaces[(int)i].MIndices[j]);
+                            ibPointer += sizeof(ushort);
+                        }
+                    }
+                }
+            }
+
+            // Build the mesh data
+            var vertexDeclaration = new VertexDeclaration(vertexElements.ToArray());
+            var vertexBufferBinding = new VertexBufferBinding(GraphicsSerializerExtensions.ToSerializableVersion(new BufferData(BufferFlags.VertexBuffer, vertexBuffer)), vertexDeclaration, (int)mesh->MNumVertices, vertexDeclaration.VertexStride, 0);
+            var indexBufferBinding = new IndexBufferBinding(GraphicsSerializerExtensions.ToSerializableVersion(new BufferData(BufferFlags.IndexBuffer, indexBuffer)), is32BitIndex, (int)nbIndices, 0);
+
+            var drawData = new MeshDraw
+            {
+                VertexBuffers = new VertexBufferBinding[] { vertexBufferBinding },
+                IndexBuffer = indexBufferBinding,
+                PrimitiveType = PrimitiveType.TriangleList,
+                DrawCount = (int)nbIndices
+            };
+
+            return new MeshInfo
+            {
+                Draw = drawData,
+                Name = meshNames[(IntPtr)mesh],
+                Bones = bones,
+                MaterialIndex = (int)mesh->MMaterialIndex,
+                HasSkinningPosition = hasSkinningPosition,
+                HasSkinningNormal = hasSkinningNormal,
+                TotalClusterCount = totalClusterCount
+            };
+        }
+
+        private void NormalizeVertexWeights(List<List<(short, float)>> controlPts, int nbBoneByVertex)
+        {
+            for (var vertexId = 0; vertexId < controlPts.Count; ++vertexId)
+            {
+                var curVertexWeights = controlPts[vertexId];
+
+                // check that one vertex has not more than 'nbBoneByVertex' associated bones
+                if (curVertexWeights.Count > nbBoneByVertex)
+                {
+                    Logger.Warning(
+                        "The input file contains vertices that are associated to more than {0} bones. In current version of the system, a single vertex can only be associated to {0} bones. Extra bones will be ignored",
+                        new ArgumentOutOfRangeException("To much bones influencing a single vertex"));
+                }
+
+                // resize the weights so that they contains exactly the number of bone weights required
+                while (curVertexWeights.Count < nbBoneByVertex)
+                {
+                    curVertexWeights.Add((0, 0));
+                }
+
+                var totalWeight = 0.0f;
+                for (var boneId = 0; boneId < nbBoneByVertex; ++boneId)
+                    totalWeight += curVertexWeights[boneId].Item2;
+
+                if (totalWeight == 0.0) // Assimp weights are positive, so in this case all weights are nulls
```

Comment:
Floating point comparison. Is it safe to compare to 0 instead of an epsilon? I assume we expect it to be exactly zero when all weights are exactly 0 and no some combination of very low weights.